core/src/main/java/com/adobe/aem/commons/assetshare/components/actions/dmdownload/DMDownload.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Asset Share Commons
+ *
+ * Copyright (C) 2017 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.adobe.aem.commons.assetshare.components.actions.dmdownload;
+
+import com.adobe.aem.commons.assetshare.content.AssetModel;
+import org.osgi.annotation.versioning.ProviderType;
+
+import java.util.Collection;
+
+@ProviderType
+public interface DMDownload {
+    Collection<AssetModel> getAssets();
+
+    String getZipFileName();
+
+    boolean isDynamicMediaEnabled();
+
+    String[] getImagePresets();
```

Comment:
WDYT about a List? I think we tend to use Lists over arrays in pretty much every ASC interface's return val?
Or a Collection like getAssets() above?

Reply:
Sure

Reply:
The type in jcr was String[]. So i just returned it as is without casting. I can change it.